src/main/java/com/michelin/ns4kafka/models/Topic.java

```diff
@@ -64,6 +64,9 @@ public static class TopicSpec {
         @Builder.Default
         @JsonSetter(nulls = Nulls.AS_EMPTY)
         private List<String> tags = new ArrayList<>();
+        @Builder.Default
+        @JsonSetter(nulls = Nulls.AS_EMPTY)
+        private String description = "";
```

Comment:
Not mandatory to set to an empty string, I thought it would be. So ok for the changes.
I also changed the enrichWithDescription to put the null value as default for topic description